pandas/tests/io/excel/test_base.py

```diff
@@ -128,96 +77,6 @@ def set_engine(self, request):
         yield
         setattr(self, func_name, old_func)
 
-    @td.skip_if_no("xlrd", "1.0.1")  # see gh-22682
-    def test_usecols_int(self, ext):
```

Comment:
you'll probably just want to split by complete classes in this PR and leave refactoring the classes to a follow-on PR.